docs/developer/developer-mode-installation.md

```diff
@@ -58,16 +58,20 @@ You can install Harvester on an [RKE2](https://docs.rke2.io/) cluster using the
     sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config
     sudo chown $(id -u):$(id -g) ~/.kube/config
     ```
+    
+:::info important
+
+Allow some time for all pods in the `kube-system` namespace to reach the `Ready` state before proceeding to the next step.
+
+If you are unable to locate the kubectl binary in `/usr/local/bin`, check `/var/lib/rancher/rke2/bin`.
+
+:::
```

Comment:
```suggestion
    :::info important

    Allow some time for all pods in the `kube-system` namespace to reach the `Ready` state before proceeding to the next step.

    If you are unable to locate the kubectl binary in `/usr/local/bin`, check `/var/lib/rancher/rke2/bin`.

    :::
```

Reply:
The current indentation is causing formatting issues.

![Screenshot 2024-08-22 at 10 30 06 PM](https://github.com/user-attachments/assets/e60e585a-8286-4733-82c7-2fea09028aec)